src/components/views/MoviesSummary.vue

```diff
@@ -1,9 +1,10 @@
 <script>
 import { defineComponent } from 'vue';
-import TheTag from '@/components/common/TheTag.vue';
+import AppTag from '@/components/common/AppTag.vue';
 import SectionTitle from '@/components/common/SectionTitle.vue';
-import MovieCard from './MovieCard.vue';
+import MovieCard from '@/components/views/MovieCard.vue';
 import TheContainer from '@/components/common/TheContainer.vue';
+import NoMatch from '@/components/views/NoMatch.vue';
```

Comment:
Again, this one doesn't seem to be used in this component's template 🤔

Reply:
I have re-reviewed all my code and found even more redundant imports -- all already deleted.